index.html

```diff
@@ -3940,99 +4037,6 @@ <h3>Denial Of Service Security Risk</h3>
     </section>
   </section>
 
-  <section> <h2>Terminology and conventions</h2>
```

Comment:
Instead of removing, we could just put these in a comment?

Reply:
It has not been removed. It was moved to the top as requested.